doc/application/application.rst

```diff
@@ -131,11 +132,11 @@ shown above in :file:`CMakeLists.txt`. The following important variables
 configure the Zephyr build system:
 
 * :makevar:`ZEPHYR_BASE`: Sets the path to the Zephyr base directory.  This is
-  usually an environment variable set by the :file:`zephyr-env.sh` script, as
-  you learned when getting started with Zephyr in
-  :ref:`getting_started_run_sample`. You can also select a specific Zephyr base
-  directory by replacing ``$ENV{ZEPHYR_BASE}`` with the specific base you'd
-  like to use instead.
+  usually an environment variable set by the :file:`zephyr-env.sh` script on
+  Linux/macOS or manually on Windows, as you learned when getting started
+  with Zephyr in :ref:`getting_started_run_sample`. You can also select a
```

Comment:
This last sentence ("You can also select ... use instead") is redundant on Windows now. Maybe reword to:

```
(You can also set :makevar:`ZEPHYR_ENV` explicitly on Unix environments if you want to.)
```

Reply:
Fixed